app.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, render_template, request, redirect, url_for, flash
+from flask_sqlalchemy import SQLAlchemy
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///database.db'
+app.config['SECRET_KEY'] = 'mysecretkey'
+db = SQLAlchemy(app)
+
+class Todo(db.Model):
```

Comment:
models should be created in a separate directory not in the main file, models/model.py